_sips/minutes/2020-03-11-minutes.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: sips
+title: SIP Meeting Minutes - March 11 2020
+
+partof: minutes
+---
+
+# Minutes
+
+The meeting took place with the following agenda:
+
+1. Quick setup, reviewing expectations for the retreat
+2. Review a list of "easy" dotty features/changes (see section below)
+3. Review enums
+4. Review creator applications
+5. Review `@infix` and `@alpha`
+6. Review dependent and polymorphic function types
+
+## Date and Location
+
+The meeting took place on the 11th March 2020 throughout the day at EPFL in Lausanne, Switzerland, and Zoom.
+
+The meeting wasn't broadcast.
+
+Minutes were taken by Dale Wijnand.
+
+## Committee Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+* Guillaume Martres ([@smarter](https://github.com/smarter)), EPFL
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
+
+## Sitting in
+
+* Jamie Thompson ([@bishabosha](https://github.com/bishabosha)), Scala Center
+* Lukas Rytz ([@lrytz](https://twitter.com/lrytz)), Lightbend
+* Nicolas Stucki ([@nicolasstucki](https://github.com/nicolasstucki)), EPFL
+* Darja Jovanovic ([@darjutak](https://github.com/darjutak)), Process Lead
+* Dale Wijnand ([@dwijnand](https://twitter.com/dwijnand)), secretary
+
+## Not present
+
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), CMU
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Triplequote
+* Josh Suereth ([@jsuereth](https://github.com/jsuereth)), independent
+* Miles Sabin ([@milessabin](https://github.com/milessabin)), independent
+
+Miles notified the committee that he wouldn't be available.  Josh, Heather, and Iulian were all unable to
+participate because of coronavirus-related travel restrictions and disruptions.
+
+## Proceedings
+
+### Expectations
+
+* Martin: The plan is still to release Scala 3 this year, at the end of December.
+* Martin: However, TASTy won't be stable until 3.1 - it won't be a 3.0 promise.
+* Martin: Ideally the features would be voted on with the following 3 outcomes
+
+* accepted
+* accepted, but put behind some kind of "preview" flag, like Java is doing
+* rejected
+* postponed to after 3.0
+
+### Review a "easy" dotty features/changes
+
+This is the list, with any notable comments added.
```

Comment:
probably explain what "easy" means. something like "We called features 'easy' if there was already substantial agreement in the pre-retreat straw poll we conducted beforehand, and if no one present felt that feature needed extended in-person discussion"